### PR TITLE
Support reading api-key from vault command

### PR DESCRIPTION
**Pull request by gandarez**
This PR makes vim to also read from vault command if present.

https://github.com/wakatime/wakatime-cli/pull/783#issuecomment-1326109831

### CODE REVIEW

File: plugin/wakatime.vim [hunk 1]
```diff
@@ -6,7 +6,7 @@
 " Website:     https://wakatime.com/
 " ============================================================================
 
-let s:VERSION = '9.0.1'
+let s:VERSION = '9.1.0'
```
Comment (alanhamlett) [inline plugin/wakatime.vim:9]: Let's not mix version bumps with code changes.

File: plugin/wakatime.vim [hunk 2]
```diff
@@ -287,6 +287,16 @@ EOF
             if s:GetIniSetting('settings', 'api_key') != '' || s:GetIniSetting('settings', 'apikey') != ''
                 let found_api_key = s:true
             endif
+
+            if !found_api_key
+                let vault_cmd = s:GetIniSetting('settings', 'api_key_vault_cmd')
+                if vault_cmd != ''
```
Comment (alanhamlett) [inline plugin/wakatime.vim:293]: `if !empty(vault_cmd)`

Comment (alanhamlett) [reply plugin/wakatime.vim:293]: https://vimhelp.org/builtin.txt.html#empty%28%29

**Review by alanhamlett — APPROVED** [review]

**Review by alanhamlett — CHANGES_REQUESTED** [review]
Just remove the version bump and it's good to merge.

File: plugin/wakatime.vim (hunk 1 above)
Comment (gandarez) [reply plugin/wakatime.vim:9]: I did it to test locally and forgot to rollback :(

File: plugin/wakatime.vim (hunk 2 above)
Comment (gandarez) [reply plugin/wakatime.vim]: Is there a guideline to use `empty` function? There are other lines of code that's not using it, like
```
if s:GetIniSetting('settings', 'api_key') != '' || s:GetIniSetting('settings', 'apikey') != ''
```

**Review by alanhamlett — APPROVED** [review]

File: plugin/wakatime.vim (hunk 2 above)
Comment (alanhamlett) [reply plugin/wakatime.vim]: That's just legacy code, we can change it to also use `empty` later.

**Pull request merged**